scripts/scene/RoomBuilder.gd

```diff
@@ -75,6 +70,7 @@ func drafting_room_motion(event: InputEventMouseMotion, offset: Vector2, zoom: V
 	draft_room(initial_tile_coords, transverse_tile_coords)
 
 func setting_door(event: InputEventMouseButton, offset: Vector2, zoom: Vector2) -> void:
+	temp_door_coords = []
```

Comment:
Need to reset this each time we restart setting the doors in case we go back in state and return.